config.example.wikimedia.yaml

```diff
@@ -450,6 +443,170 @@ templates:
+                  uri: /{domain}/sys/graphoid/v1/handler/{format}
+                  headers:
+                    cache-control: '{cache-control}'
+                    x-resource-location: '{$.calculate_hash.body}'
+                  body: '{$.request.body}'
+            - store_request:
+                request:
+                  method: put
+                  uri: /{domain}/sys/post_data/graphoid.request/
+                  body: '{$.request.body}'
+            - return_content:
+                return:
+                  status: 200
+                  headers: '{$.load_data.headers}'
+                  body: '{$.load_data.body}'
+      /{module:graphoid}/v1/preview/{format}:
+        post:
+          x-request-handler:
+            - calculate_hash:
+                request:
+                  method: put
+                  uri: /{domain}/sys/post_data/graphoid.request/hash
+                  body: '{$$.merge($.request.body, {"x-restbase-mode": "preview"})}'
+            - handle_req:
+                request:
+                  method: post
+                  uri: /{domain}/sys/graphoid/v1/handler/{$.request.params.format}
+                  headers:
+                    max-age: 3600
+                    x-resource-location: '{$.calculate_hash.body}'
+                    content-type: application/json
+                  body: '{$.request.body}'
+      /{module:graphoid}/v1/request/json/{hash}:
+        get:
+          x-request-handler:
+            - check_request_storage:
+                request:
+                  method: get
+                  uri: /{domain}/sys/post_data/graphoid.request/{hash}
+                return:
+                  status: 200
+                  headers:
+                    content-type: application/json
+                  body: '{$.check_request_storage.body}'
+      /{module:graphoid}/v1/request/{format}/{hash}:
+        get:
+          x-request-handler:
+            - check_storage:
+                request:
+                  method: get
+                  uri: /{domain}/sys/key_value/graphoid.{$.request.params.format}/{hash}
+                  headers:
+                    cache-control: '{cache-control}'
+                return_if:
+                  status: '2xx'
+                return:
+                  status: 200
+                  headers: '{$.check_storage.headers}'
+                  body: '{$.check_storage.body}'
+                catch:
+                  status: 404
+            - load_request:
+                request:
+                  uri: /{domain}/sys/post_data/graphoid.request/{hash}
+            - handle_req:
+                request:
+                  method: post
+                  uri: /{domain}/sys/graphoid/v1/handler/{format}
+                  headers:
+                    cache-control: '{cache-control}'
+                    x-resource-location: '{$.request.params.hash}'
+                    content-type: application/json
+                  body: '{$.load_request.body}'
```

Comment:
This could also be where we make the graphoid request & store the response. We could then skip the entire block below, simplifying the overall stanza.